src/Command/SearchSettingsCommand.php

```diff
@@ -2,22 +2,37 @@
 
 namespace Algolia\SearchBundle\Command;
 
-use Algolia\SearchBundle\Settings\SettingsManagerInterface;
+use Algolia\SearchBundle\Settings\SettingsManager;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
+/**
+ * @internal
+ */
 abstract class SearchSettingsCommand extends Command
 {
+    /**
+     * @var SettingsManager
+     */
     protected $settingsManager;
 
-    public function __construct(SettingsManagerInterface $settingsManager)
+    /**
+     * @param SettingsManager $settingsManager
+     */
+    public function __construct(SettingsManager $settingsManager)
     {
         $this->settingsManager = $settingsManager;
 
         parent::__construct();
     }
 
+    /**
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return int|null
```

Comment:
Not PHP fluent so my comment might not be relevant but:

Why does it say `@return int|null` and the method always returns null? Shouldn't the method be `void`?